src/createWidgetBase.ts

```diff
@@ -143,6 +149,24 @@ function manageDetachedChildren(instance: Widget<WidgetProperties>): void {
 	});
 }
 
+function bindFunctionProperties(instance: Widget<WidgetProperties>, properties: WidgetProperties) {
+	Object.keys(properties).forEach((propertyKey) => {
+		const func = properties[propertyKey];
+		const bind = properties.bind;
+
+		if (typeof func === 'function') {
```

Comment:
I guess the logic of (if it's a function do this, if it's a string do that)... that said, I realize it's pretty specific to what you're doing here, so just ignore me. :)